src/main/java/de/qabel/desktop/DesktopClient.java

```diff
@@ -259,6 +261,8 @@ public void run() {
         });
         primaryStage.show();
 
+        runtimeConfiguration.loadAboutFiles();
```

Comment:
did I mention that you could add this to the translation files?
like `ui.properties`? By terminating a line with `\` you can add multiline content. Then the file loading thing would have been done by magic ;)